xml/admin_gui_dashboard.xml

```diff
@@ -2621,6 +2621,12 @@ Please consult the documentation on how to configure and enable the Object Gatew
     port. By default, the currently active &mgr; that hosts the dashboard binds
     to TCP port 8443 (or 8080 when SSL is disabled).
    </para>
+   <note>
+     <para>
+       For more information on firewall settings for &ceph;, see
```

Comment:
I would lead into that note with a hint about "If a firewall is enabled on the hosts running Ceph Mgr (and thus Ceph Dashboard), you may have to change it's configuration to enable access to these ports."

Reply:
Verbosity is my velocity